.gitignore

```diff
@@ -55,6 +55,8 @@ coverage.xml
 *.py,cover
 .hypothesis/
 .pytest_cache/
+tests/regression_test_results.json
+tests/regression_test_baselines.json
```

Comment:
I don't get this. Why do we first put it in the gitignore just to then force add it?

Reply:
I agree this appears a bit odd, but I want to prevent users from pushing an updated baseline file from their local machine (which is why I have added it to the gitignore). The only way baselines should be updated is through github actions (to ensure regression tests are run in a consistent environment). That is why the github actions uses -f, while for the user changes to these files are ignored.

Reply:
Does that make sense?

Reply:
Ah, got it, yes makes sense!